.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+> Hi there! Thanks for submitting a pull request to Concourse! To help us review your PR,
+> please fill in the following information.
+> Feel free to delete any sections not applicable to your pull request.
+
+# Existing Issue
+> Is there an existing issue related to this PR? Fill in the issue number as follows: Fixes #1234.
+
+Fixes # .
+
+# Why do we need this PR?
+> No issue number? Please give us a few lines of context describing the need for this PR.
+
+
+# Changes proposed in this pull request
+> What are the changes included in this PR? Feel free to add as many lines as needed below.
+
+* 
+* 
+*
+
+# Contributor Checklist
+> Are the following items included as part of this PR?
+- [ ] Unit tests
+- [ ] Integration tests (if applicable)
+- [ ] Updated documentation
+- [ ] Updated release notes
```

Comment:
Can we link to where they would update the documentation or the release notes?
I am sorry forgot to mention that the first time.

Reply:
done